pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -29,12 +33,27 @@ public class CreateClinicalDummyKitRoute implements Route {
     private static final String USER_ID = "MERCURY";
     private static String CLINICAL_KIT_REALM = "CLINICAL_KIT_REALM";
     private static String CLINICAL_KIT_PREFIX = "CLINICALKIT_";
-    private final String ffpeUser = "ffpe-dummy-kit-creator";
+    private static final String ffpeUser = "ffpe-dummy-kit-creator";
     private final String ffpe = "ffpe";
     private final String ffpeScroll = "ffpe-scroll";
     private final String ffpeSection = "ffpe-section";
     private int realm;
 
+    public static void addCollaboratorSampleId(String tissueId, DDPInstance ddpInstance, String ddpParticipantId, String shortId) {
+        String collaboratorParticipantId = KitRequestShipping
+                .getCollaboratorParticipantId(ddpInstance.getBaseUrl(), ddpInstance.getDdpInstanceId(), ddpInstance.isMigratedDDP(),
+                        ddpInstance.getCollaboratorIdPrefix(), ddpParticipantId, shortId, "4");
+        updateTissue(tissueId, collaboratorParticipantId + "_T1");
+
+    }
+
```

Comment:
Since it is now needed to have collaborator sample id in the test responses for James, I am adding a collaborator Sample id to the dummy tissue we insert